packages/strapi-plugin-content-type-builder/admin/src/containers/Form/index.js

```diff
@@ -446,17 +446,16 @@ export class Form extends React.Component { // eslint-disable-line react/prefer-
           toAdd = 0;
           break;
       }
-  
+
       this.setState(prevState => ({ nodeToFocus: prevState.nodeToFocus + toAdd }));
     }
   }
 
-  handleSubmit = (e, redirectToChoose = true) => {
+  handleSubmit = (e, redirectToChoose = false) => {
```

Comment:
I'm wondering why you changed the redirectToChoose param?

Reply:
Yes it shouldn't i had issue with syncing my fork, put code back.